modules/tensor_mechanics/include/materials/LAROMANCEStressUpdateBase.h

```diff
@@ -58,12 +58,13 @@ class LAROMANCEStressUpdateBaseTempl : public RadialReturnCreepStressUpdateBaseT
   virtual Real computeTimeStepLimit() override;
 
   /// Enum to error, warn, ignore, or extrapolate if input is outside of window of applicability
-  enum class WindowFailure
```

Comment:
Why all the changes?

Reply:
Look at the answer below, but it is to get the indexing right.